script.motr/addon.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.motr" name="Movies On The Run" version="19.0.3" provider-name="Lars Werner">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="main.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Updated for Kodi Matrix!
```

Comment:
you have summary and description swapped :)

Reply:
:) Didn't noticed, fixed now, thanks.